### PR TITLE
add mixed chinese rule

### PR DESCRIPTION
**Pull request by yshrkt**
- Adds mixed chinese rule for issue #8

### CODE REVIEW

File: Sources/L10nLintFramework/Rules/Extensions/MixedChineseRule+SimplifiedCharacters.swift [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
...\x{31297}\\x{31298}\\x{31299}\\x{3129A}\\x{3129B}\\x{3129C}\\x{3129D}\\x{3129E}\\x{3129F}\\x{312A0}\\x{312A1}\\x{312A2}\\x{312A3}\\x{312A4}\\x{312A5}\\x{312A6}\\x{312A7}\\x{312A8}\\x{312A9}\\x{312AA}\\x{312AB}\\x{312AC}\\x{312AD}\\x{312AE}\\x{312AF}\\x{312B0}\\x{312B1}\\x{312B2}\\x{312B3}\\x{312B4}\\x{312B5}\\x{312BA}\\x{312BB}\\x{312BC}\\x{312BD}\\x{312C2}\\x{312C4}\\x{312C5}\\x{312C6}\\x{312C7}\\x{312C8}\\x{312C9}\\x{312CA}\\x{312CB}\\x{312CC}\\x{312CD}\\x{312CE}\\x{312D0}\\x{312D1}\\x{312D3}\\x{312D4}\\x{312D5}\\x{312D6}\\x{312D7}\\x{312D8}\\x{312D9}\\x{312DA}\\x{312DC}\\x{312DD}\\x{312DF}\\x{312E0}\\x{312E1}\\x{312E2}\\x{312E3}\\x{312E4}\\x{312E5}\\x{312E6}\\x{312E8}\\x{312EA}\\x{312EB}\\x{312EC}\\x{312ED}\\x{312EE}\\x{312F1}\\x{312F4}\\x{312F6}\\x{312FE}\\x{312FF}\\x{31300}\\x{31301}\\x{31303}\\x{31304}\\x{31305}\\x{31306}\\x{31307}\\x{31308}\\x{31309}\\x{3130A}\\x{3130F}\\x{31315}\\x{31316}\\x{31317}\\x{31318}\\x{31319}\\x{3132B}\\x{3132C}\\x{3132D}\\x{3132E}\\x{3132F}\\x{31330}\\x{31331}\\x{31332}\\x{31333}\\x{31334}\\x{31335}\\x{31336}\\x{31337}\\x{31338}\\x{31339}\\x{3133A}\\x{3133C}\\x{3133D}\\x{31341}\\x{31342}\\x{31344}\\x{31345}\\x{31346}\\x{31347}\\x{31348}\\x{31349}"
```
Comment (s2mr) [inline Sources/L10nLintFramework/Rules/Extensions/MixedChineseRule+SimplifiedCharacters.swift:3]: Can you please comment on the citation?

File: Tests/L10nLintFrameworkTests/Resources/Fixtures/Localizables6/Base.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// MARK: KeyValueExtraSpaceRule
```
Comment (s2mr) [inline Tests/L10nLintFrameworkTests/Resources/Fixtures/Localizables6/Base.lproj/Localizable.strings:1]: `KeyValueExtraSpaceRule` is not suitable.
Please change to MixedChineseRule.

File: Sources/L10nLintFramework/Rules/MixedChineseRule.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+struct MixedChineseRule: Rule {
+    static let description: RuleDescription = .init(identifier: "mixed_chinese", name: "Mixed Chinese", description: "The mixing of traditional and simplified chinese characters should be resolved.")
+
+    init() {}
+
+    func validate(baseProject: LocalizedProject, project: LocalizedProject) throws -> [StyleViolation] {
+        print("project name: \(project.name)")
```
Comment (s2mr) [inline Sources/L10nLintFramework/Rules/MixedChineseRule.swift:9]: Remove this. 🙏🏻

**Review by s2mr — COMMENTED** [review]
Thanks for contribution!
I commented, please check it.

File: Sources/L10nLintFramework/Rules/Extensions/MixedChineseRule+SimplifiedCharacters.swift (hunk 1 above)
Comment (yshrkt) [reply Sources/L10nLintFramework/Rules/Extensions/MixedChineseRule+SimplifiedCharacters.swift:7]: Added comment.

File: Tests/L10nLintFrameworkTests/Resources/Fixtures/Localizables6/Base.lproj/Localizable.strings (hunk 2 above)
Comment (yshrkt) [reply Tests/L10nLintFrameworkTests/Resources/Fixtures/Localizables6/Base.lproj/Localizable.strings]: Changed it.

File: Sources/L10nLintFramework/Rules/MixedChineseRule.swift (hunk 3 above)
Comment (yshrkt) [reply Sources/L10nLintFramework/Rules/MixedChineseRule.swift]: Sorry, removed debug print.

**Comment by s2mr** [comment]
Thank you.
Merge it.

**Pull request merged**